hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCommitMetadata.java

```diff
@@ -116,19 +116,16 @@ public String toJsonString() throws IOException {
       log.info("partition path is null for " + partitionToWriteStats.get(null));
       partitionToWriteStats.remove(null);
     }
-    ObjectMapper mapper = new ObjectMapper();
-    mapper.setVisibility(JsonMethod.FIELD, JsonAutoDetect.Visibility.ANY);
-    return mapper.defaultPrettyPrintingWriter().writeValueAsString(this);
+    ObjectMapper mapper = getObjectMapper();
+    return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(this);
   }
 
   public static HoodieCommitMetadata fromJsonString(String jsonStr) throws IOException {
     if (jsonStr == null || jsonStr.isEmpty()) {
       // For empty commit file (no data or somethings bad happen).
       return new HoodieCommitMetadata();
     }
-    ObjectMapper mapper = new ObjectMapper();
-    mapper.configure(Feature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    mapper.setVisibility(JsonMethod.FIELD, JsonAutoDetect.Visibility.ANY);
+    ObjectMapper mapper = getObjectMapper();
```

Comment:
why not just `getObjectMapper().readValue(....)` ?

Reply:
no problem I'm going to inline it